### PR TITLE
Separates actions.dmi into multiple files

### PR DESCRIPTION
**Pull request by ghost**
`actions.dmi` is way too cluttered and messy, and with inhands getting the axe I decided to fix this, too. I separated the major categories of actions into their own files, like antagonist-specific actions and spells. A full list of categories is found here:

* `actions_AI.dmi` contains AI-specific actions;
* `actions_clockcult.dmi` contains clockcult-specific actions, and scripture quickbind icons;
* `actions_cult.dmi` contains cult-specific actions as well as sintouch;
* `actions_flightsuit.dmi` contains flightsuit and flightpack actions;
* `actions_items.dmi` contains minor item-specific actions such as the hierophant club and jetboots;
* `actions_mecha.dmi` contains actions related to mechs;
* `actions_minor_antag.dmi` contains antag-related actions with no other spot, like devils/abductors;
* `actions_revenant.dmi` contains revenant ability icons;
* `actions_silicon.dmi` contains minor silicon and console icons, like pAI and xenobiology consoles;
* `actions_slime.dmi` contains slime actions;
* `actions_spells.dmi` contains icons for all spells;
* `actions_xeno.dmi` contains icons for all xeno actions; and
* `backgrounds.dmi` contains backgrounds for the action buttons.

### CODE REVIEW

**Comment by Armhulen** [comment]
thanks 4 hard work

**Comment by Xhuis** [comment]
This is currently what the new `actions.dmi` looks like:



I'm unsure how I want to separate what's left.

**Comment by ShizCalev** [comment]
silicon.dmi (camera_off, camera_jump, drone_vision, ai_*, pai_*, ect)
building.dmi (build*)

**Review by Cyberboss — APPROVED** [review]

File: code/datums/action.dm [hunk 1]
```diff
@@ -10,12 +10,13 @@
 	var/check_flags = 0
 	var/processing = FALSE
 	var/obj/screen/movable/action_button/button = null
-	var/button_icon = 'icons/mob/actions.dmi'
-	var/background_icon_state = ACTION_BUTTON_DEFAULT_BACKGROUND
 	var/buttontooltipstyle = ""
 
-	var/icon_icon = 'icons/mob/actions.dmi'
-	var/button_icon_state = "default"
+	var/button_icon = 'icons/mob/actions/backgrounds.dmi' //This is the file for the BACKGROUND icon
```
Comment (AnturK) [inline code/datums/action.dm]: Could rename these, not sure what was i thinking when naming these.

**Comment by Xhuis** [comment]
The actions have been purged. `actions.dmi` now has four actions: vote, the language menu, and the Show/Hide icon for the action button show/hide button; those make sense to stay there, so. Here are the rest:

* `actions_AI.dmi` contains AI-specific actions;
* `actions_clockcult.dmi' contains clockcult-specific actions, and scripture quickbind icons;
* `actions_cult.dmi` contains cult-specific actions as well as sintouch;
* `actions_flightsuit.dmi` contains flightsuit and flightpack actions;
* `actions_items.dmi` contains minor item-specific actions such as the hierophant club and jetboots;
* `actions_mecha.dmi` contains actions related to mechs;
* `actions_minor_antag.dmi` contains antag-related actions with no other spot, like bloodcrawl;
* `actions_revenant.dmi` contains revenant ability icons;
* `actions_silicon.dmi` contains minor silicon and console icons, like pAI and xenobiology consoles;
* `actions_slime.dmi` contains slime actions;
* `actions_spells.dmi` contains icons for all spells;
* `actions_xeno.dmi` contains icons for all xeno actions; and
* `backgrounds.dmi` contains backgrounds for the action buttons.

This should be ready to go! Keep an eye out for missing button icons in the event that I missed something, which is quite possible given that I replaced nearly every action button icon in the game.

**Pull request merged**